testing-harness/fixtureLoader.js

```diff
@@ -2,14 +2,16 @@ const fs = require('fs');
 const path = require('path');
 
 /**
- * Load all ELM files in build directory as JSON
+ * Load a directory of JSON files
  *
- * @returns {Array} array of ELM JSON objects
+ * @param {string} currentDir current path of file trying to do the loading
+ * @param {string} relativePathToDir relative path from the caller of where directory is located
+ * @returns {Array} array of JSON parsed fixtures
  */
-function loadELM() {
-  const p = path.join(__dirname, '../output-elm');
-  const elmFiles = fs.readdirSync(p).filter((f) => path.extname(f) === '.json');
-  return elmFiles.map((f) => JSON.parse(fs.readFileSync(path.join(p, f), 'utf8')));
+function loadJSONFromDirectory(currentDir, relativePathToDir) {
+  const p = path.resolve(path.join(currentDir, relativePathToDir));
```

Comment:
This is building a bit off the comment I had on #15, but I think it might make sense to move this `path.join` and `path.resolve` out of this function and require someone to pass in an absolute path to the function. I think this might help us out when this is removed, although I'm open to other's opinions, including @Dtphelan1 since this sort of stemmed from your PR.

Reply:
The code for the `loadJSONFromDirectory` function was a combination of the previous `loadELM` and `loadValueSet` functions and the `loadJSONFixture` function. I'm not sure who originally wrote those functions but I would be interested to hear their opinion

Reply:
I think that might have been @mgramigna so I'm tagging him as well for good measure 😀

Reply:
I looked at this a little more, and I think I prefer to pass the absolute path into the `loadJSON` functions. This way we know we are getting a full path that resolves. I think that might be more clear than passing in two path parts as two arguments that we'll eventually join. It also lets us require that the person using the function constructs a path correctly, and doesn't assume they've given us enough pieces that we can.

Still happy to hear others' thoughts on this.

Reply:
That makes sense to me. I'll go make that change now